cli/command/container/opts.go

```diff
@@ -850,8 +850,8 @@ func parseLoggingOpts(loggingDriver string, loggingOpts []string) (map[string]st
 	return loggingOptsMap, nil
 }
 
-// takes a local seccomp daemon, reads the file contents for sending to the daemon
-func parseSecurityOpts(securityOpts []string) ([]string, error) {
+// ParseSecurityOpts takes a local seccomp daemon, reads the file contents for sending to the daemon
+func ParseSecurityOpts(securityOpts []string, rel func(path string) (string, error)) ([]string, error) {
```

Comment:
The motivation for exporting this came from a Compose PR that wanted to make changes to this method because it mutates the input.

Can't comment directly on the line (873 in the before diff):
```go
securityOpts[key] = fmt.Sprintf("seccomp=%s", b.Bytes())
```

I think it'd be good to create a new slice to return from this method rather than mutating the input, particularly now that it's exported.